Cargo.toml

```diff
@@ -2,5 +2,14 @@
 members = ["spotify_player", "lyric_finder"]
 resolver = "2"
 
+[workspace.package]
+rust-version = "1.74"
```

Comment:
will it return a compile error if you have an older version than this one?

Reply:
As it is, yes, I get the following:
```
error: package `clap_lex v0.7.0` cannot be built because it requires rustc 1.74 or newer, while the currently active rustc version is 1.73.0
Either upgrade to rustc 1.74 or newer, or use
cargo update -p clap_lex@0.7.0 --precise ver
where `ver` is the latest version of `clap_lex` supporting rustc 1.73.0
```

I don't know if there are dependencies unrelated to clap that also ask for >=1.74